src/_codelabs/index.md

```diff
@@ -6,23 +6,82 @@ permalink: /codelabs
 toc: false
 ---
 
-See the latest Dart codelabs on the Google Developers site:
+The latest Dart codelabs are on the Google Developers site.
 
-**General**
-* [Intro to Dart for Java Developers](https://codelabs.developers.google.com/codelabs/from-java-to-dart/)
+## General
 
-**Flutter**
-* [Building Beautiful UIs with Flutter](https://codelabs.developers.google.com/codelabs/flutter/)
-* [Firebase for Flutter](https://codelabs.developers.google.com/codelabs/flutter-firebase/)
+<div class="item-with-pic" markdown="1">
+<img src="https://webdev.dartlang.org/codelabs/images/from-java-to-dart.png"
+     width="150px"
+     alt="Bicycle image from codelab">
+<div class="details" markdown="1">
+#### [Intro to Dart for Java Developers](https://codelabs.developers.google.com/codelabs/from-java-to-dart/)
+{: .title}
```

Comment:
I believe that would cause problems in the books page, which uses two headings (h3 & h4).